include/functions.h

```diff
@@ -49,6 +49,7 @@ void step_game_loop(void);
 s32 resume_all_group(s32 groupFlags);
 f32 length2D(f32 x, f32 y);
 void player_input_to_move_vector(f32* angle, f32* magnitude);
+void game_input_to_move_vector(f32*, f32*);
```

Comment:
This will produce a warning ('not a prototype') - give the arguments names (x and y)?

Reply:
The "not a prototype" warning comes when the arguments to a function are left completely unspecified (no `void`, just `()`). You're allowed to (and should) have unnamed arguments in function declarations if we're just copy-pasting them to headers to get stuff to match.

If we start putting arg names in function declarations whenever we make them, we're going to have multiple places we need to tend to when we inevitably rename arguments. Fixing arg names in multiple places is going to be a pain., so they're best only included in the function declaration if the declaration is there to serve as documentation and not just a means of getting the compiler to create the correct codegen.

Reply:
[It does emit a warning.](https://decomp.me/scratch/hhzA2)

Reply:
Oh, that's true. I thought this did emit a warning. For the vast majority of declarations added in this PR (and, probably, future PRs), we already know what the argument names should be thanks to clover's lib files. So we should use them.